src/components/PasswordStrengthMeter/index.tsx

```diff
@@ -93,11 +93,15 @@ const PasswordStrengthMeter = ({
 
   return (
     <div title={title} role="alert" aria-live="polite">
-      <StyledTitle variant="bodyB" color={theme.colors.neutral.text}>
+      <StyledTitle variant="bodySmallStrong" as="p">
         {title}
       </StyledTitle>
 
-      <StyledStrength as="span" variant="bodyB" color={strength[score]?.color}>
+      <StyledStrength
+        as="span"
+        variant="bodySmallStrong"
+        strength={strength[score]}
+      >
```

Comment:
```suggestion
      <StyledStrength
        as="span"
        variant="bodySmall"
        strength={strength[score]}
      >
```

That's what I meant :p